adapters/tlsio_wolfssl.c

```diff
@@ -892,6 +901,10 @@ int tlsio_wolfssl_send(CONCRETE_IO_HANDLE tls_io, const void* buffer, size_t siz
             {
                 result = 0;
             }
+
+            // remove on send complete and callback context
+            memset((void*)&tls_io_instance->on_send_complete, 0, sizeof(tls_io_instance->on_send_complete));
```

Comment:
isn't this just setting the pointer to NULL?

Reply:
It is, but I wanted to mimic the code that was already in tlsio_mbedtls.c doing its own reset (albeit of a struct containing the function pointer and the callback context).  Maybe it's not worth it.